sdk/healthinsights/azure-health-insights-cancerprofiling/README.md

```diff
@@ -0,0 +1,131 @@
+# Azure Azure Health Insights Cancer Profiling client library for Java
+<!--
+[Health Insights](https://review.learn.microsoft.com/en-us/azure/cognitive-services/health-decision-support/overview?branch=main) is an Azure Applied AI Service built with the Azure Cognitive Services Framework, that leverages multiple Cognitive Services, Healthcare API services and other Azure resources.
+The [Cancer Profiling model](https://review.learn.microsoft.com/en-us/azure/cognitive-services/health-decision-support/oncophenotype/overview?branch=main) receives clinical records of oncology patients and outputs cancer staging, such as clinical stage TNM categories and pathologic stage TNM categories as well as tumor site, histology.
+-->
+
+## Getting started
+
+### Prerequisites
+
+- A [Java Development Kit (JDK)][jdk] with version 8 or above
```

Comment:
jdk link is not attached: 
![Uploading image.png…]()

Reply:
thanks